### ISSUE TITLE
Show progress for deleteing a directory

### ISSUE THREAD

**Issue by Milo123459** [microsoft/vscode#131815]
A simple little progress bar where it says "Deleting XYZ" should be like (Deleting XYZ (15%))

Simple, but effective.

**Comment by bpasero** [comment]
No, needs support from Electron: https://github.com/electron/electron/blob/13-x-y/docs/api/shell.md#shelltrashitempath

**Comment by Milo123459** [comment]
Is there a feature request for it? I genuinely want this as I don't know how long it's going to take to delete a directory.

**Comment by Milo123459** [comment]
^ I have created an issue upstream. I am going to look into implementing this at electron, then I (hopefully) will write a PR for it when that releases stablely.

**Comment by lramos15** [comment]
Not planned for electron so unfortunately not planned for us

https://github.com/electron/electron/issues/31028
